src/content/en/tools/chrome-devtools/profile/network-performance/understanding-resource-timing.markdown

```diff
@@ -0,0 +1,222 @@
+---
+layout: shared/narrow
+title: "Understanding Resource Timing"
+description: "It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues."
+published_on: 2016-02-04
+order: 2
+authors:
+  - jonathangarbee
+translation_priority: 0
+key-takeaways:
+  timing:
+    - "Understand the phases of resource timing."
+    - "Know what each phase provides to the Resource Timing API."
+    - "Realize different indicators of performance problems in the timeline graph. Such as series of transparent bars or large green chunks."
+notes:
+  note-tbd:
+    - "TBD note."
+---
+
+<p class="intro">
+  It is crucial to understand the phases in which resources are gathered over the network. This is the foundation for fixing load issues.
+</p>
+
+{% include shared/toc.liquid %}
+
+{% include shared/takeaway.liquid list=page.key-takeaways.emulate-network %}
+
+All network requests are considered resources.
+As they are retrieved over the network, resources have distinct lifecycles expressed in terms of resource timing.
+The network panel uses the same [Resource Timing API](http://www.w3.org/TR/resource-timing) that is available to application developers.
```

Comment:
s/network panel/Network Panel/

Capitalize "Network Panel" throughout the document.